x-pack/dockerlogbeat/magefile.go

```diff
@@ -147,11 +146,14 @@ func Package() {
 // Release builds a "release" tarball that can be used later with `docker plugin create`
 func Release() error {
 	mg.Deps(Build)
-	name, err := getPluginName()
+
+	version, err := mage.BeatQualifiedVersion()
 	if err != nil {
-		return err
+		return errors.Wrap(err, "error getting beats version")
 	}
 
+	tarballName := fmt.Sprintf("%s-%s-%s.tar.gz", name, version, "docker-build-context")
```

Comment:
Can you use `docker-plugin` as the **classifier** please instead of `docker-build-context`.
The `docker-build-context` classifier is used for the artifacts we publish in https://github.com/elastic/dockerfiles where we have all the files (context) required for building an image via the `docker build` command.

Thanks

Reply:
Sure! I was a little confused by what classifier was appropriate, heh